### DISCUSSION TITLE
sFTP ssh version

### DISCUSSION THREAD

**Discussion by rutledj** [nextgenhealthcare/connect#5957] [Q&A]
Does anyone know what version of ssh that is used for sftp in mirth (version 4.4)?

**Comment by jonbartels** [comment]
See: https://github.com/nextgenhealthcare/connect/issues/5608

The supported SSH version would be whatever versions existed when jSch last released in 2018. That should be SSH2.

Are you seeing a specific problem with an SSH or SFTP connection from Mirth?
